android/tooling/rib-intellij-plugin/src/main/java/com/uber/presidio/intellij_plugin/action/rib/GenerateRibAction.java

```diff
@@ -24,19 +24,19 @@
  */
 public class GenerateRibAction extends GenerateAction implements GenerateRibDialog.Listener {
 
-  @Override
-  public void actionPerformed(AnActionEvent anActionEvent) {
-    GenerateRibDialog dialog = new GenerateRibDialog(this);
-    dialog.show();
-  }
+    @Override
```

Comment:
I just realized that you're adding new spaces. The Uber code style is actually to use "two spaces" everywhere. Our code style is basically https://google.github.io/styleguide/javaguide.html .

I might have left the templates still for four spaces (I shouldn't have done that) because the templates are slightly different internally and externally. We incorporate some more proprietary code into the internal version of the templates.

Reply:
Sure! :)